spread.yaml

```diff
@@ -863,22 +863,12 @@ suites:
             dpkg -i "$SPREAD_PATH"/../snapd_*.deb
         prepare-each: |
             tests.backup prepare
-
-            #shellcheck source=tests/lib/nested.sh
-            . "$TESTSLIB/nested.sh"
-            nested_prepare_env
+            "$TESTSTOOLS"/nested-state prepare
         restore-each: |
-            #shellcheck source=tests/lib/nested.sh
-            . "$TESTSLIB/nested.sh"
-            nested_destroy_vm
-            nested_cleanup_env
-
+            "$TESTSTOOLS"/nested-state remove-vm
+            "$TESTSTOOLS"/nested-state restore
             tests.backup restore
         restore: |
-            #shellcheck source=tests/lib/nested.sh
-            . "$TESTSLIB/nested.sh"
-            nested_cleanup_env
```

Comment:
This is not needed because restore-each does it right?

Reply:
There's a call to `nested-state restore` in other suites, so it's unclear whether we need it or not. Do we?

Reply:
The idea is that in the manual suite we create/restore the whole environment for each test. So as for each test we do "$TESTSTOOLS"/nested-state restore, then it not needed to do it when restoring the suite.
For other suites, on each test restore we do "$TESTSTOOLS"/nested-state remove-vm, so then when we restore the suite we need to do "$TESTSTOOLS"/nested-state restore.